tests/integration/docusaurus/expectations/advanced/multi_batch_rule_based_profiler_example.py

```diff
@@ -112,7 +113,10 @@
     data_context=data_context,
 )
 
-suite = rule_based_profiler.run(expectation_suite_name="test_suite_name")
```

Comment:
An unrelated broken test creeped in somehow...

Reply:
Because `docusaurus` tests are no longer blocking...